graphlearn_torch/python/partition/base.py

```diff
@@ -116,6 +116,58 @@ def save_feature_partition(
     torch.save(feature_partition.cache_feats, os.path.join(subdir, 'cache_feats.pt'))
     torch.save(feature_partition.cache_ids, os.path.join(subdir, 'cache_ids.pt'))
 
+def save_feature_partition(
```

Comment:
Now there are two duplicate methods?

Reply:
removed the one that stores feature as pt file